actor/src/lib.rs

```diff
@@ -491,7 +491,7 @@ where
     M::Result: Send,
 {
     async fn handle(self, sender: oneshot::Sender<M::Result>) {
-        drop(sender.send(self));
+        let _send_event_result = sender.send(self);
```

Comment:
I agree that ignoring a result is not the right approach. We did this because we don't want deadlock the actor, but considering work done by @s8sato and ongoing work by you and @SamHSmith I think we might add an error handling mechanism that doesn't block.

Reply:
Ok, leave it as is for now.

Reply:
I don't understand what is gained with this variable binding?

Reply:
Don't have strong opinion on this, but i think by this binding we indicate that we don't care about `Result` returned from `sender.send(self)`.
We can't just write because of [unused_must_use](https://doc.rust-lang.org/stable/nightly-rustc/rustc_lint/unused/static.UNUSED_MUST_USE.html):
```rust
sender.send(self);
```